### PR TITLE
Login signup page

### PR DESCRIPTION
**Pull request by Aazam10**
Hi I have made login and signup page for ecommerce site. I have not made it completely responsive will try to do once I am done with P0 screens .Please review

CSB for login: https://ir3wp.csb.app/login.html
CSB for signup: https://ir3wp.csb.app/signup.html

### CODE REVIEW

File: authorization/auth.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+.auth-card {
+  /* width: max-content;
+  max-width: 90%; */
+  width: 30%;
+  min-width: 300px;
+  padding: 2rem;
+  /* border: 1px solid black; */
+  margin: 0 auto;
+
+  box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
```
Comment (ruchiragoswami) [inline authorization/auth.css:17]: Just a suggestion, I use box-shadow in many components so for reusability, I make a variable for it. 
example, --box-shadow: rgba(100, 100, 111, 0.2) ;

**Review by ruchiragoswami — APPROVED** [review]
Good work Aazam, 
I personally like using 'section' tags  also, so its not just 'div'

**Review by NagaSatyaGeeth — APPROVED** [review]

File: authorization/auth.css (hunk 1 above)
Comment (Aazam10) [reply authorization/auth.css:17]: I have been using different box shadows for different components according to situation that is why not adding it to a variable.If same repeats many times ill add that going forward

**Comment by Aazam10** [comment]
Will make sure to use semantic html more going forward. Thanks ruchi

**Pull request merged**